community/teams/security.tt

```diff
@@ -77,7 +77,7 @@
     <li>The issues tagged "Security" on <a
         href="https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%221.severity%3A+security%22">GitHub/NixOS</a>
     </li>
-    <li>The #nixos-security IRC channel on FreeNode
+    <li>The <a href="https://app.element.io/#/room/#security:nixos.org">#security</a> room on Matrix
```

Comment:
```suggestion
    <li>The <a href="https://matrix.to/#/#security:nixos.org">#security:nixos.org</a> room on Matrix
```

AIUI, matrix.to links can be opened in whatever client someone is using, rather than going straight to element web.